examples/settings.gradle

```diff
@@ -0,0 +1,2 @@
+include ':android-simple'
+include ':android-activity-graphs'
```

Comment:
Not tested but would it work with the following? (annotation processing still needs to be [enabled manually in the IDE](https://github.com/tbroyer/gradle-apt-plugin#usage-with-ides); and in Android Studio it's, er…, [hidden](https://stackoverflow.com/questions/38642712/enable-annotation-processors-option-in-android-studio-2-2/38698186#38698186))

…or move the `gradlew` (et al.) and `buildscript { … classpath 'com.android.tools.build:gradle:2.2.3' … }` down into each `android-*` project? (i.e. only build the Android samples using Gradle, which should also allows them to be imported into Android Studio)

```gradle
// examples/simple/build.gradle
plugins {
  id "net.ltgt.apt" version "0.9"
}
apply plugin: 'java'
apply plugin: 'idea'

dependencies {
    compile "com.google.dagger:dagger:$dagger_version"
    apt "com.google.dagger:dagger-compiler:$dagger_version"
}
```